File: be/dotnet/StarSecurity/Domain/Entities/Offer.cs
using Domain.Entities.Common;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Domain.Entities
{
	public class Offer : Entity
	{
		[Required(ErrorMessage = "{PropertyName} is required")]
		public decimal TotalPayment { get; set; }
		[Required(ErrorMessage = "{PropertyName} is required")]
		public DateTime StartDate { get; set; }
		[Required(ErrorMessage = "{PropertyName} is required")]
		public DateTime EndDate { get; set; }
		[Required(ErrorMessage = "{PropertyName} is required")]
		public string ProvideService { get; set; }

		//n-1 with Employee
		[Required(ErrorMessage = "{PropertyName} is required")]
		public Guid EmployeeId { get; set; }
		[ForeignKey("EmployeeId")]
		public virtual Employee? Employee { get; set; }
		//n-1 with Client
		[Required(ErrorMessage = "{PropertyName} is required")]
		public Guid ClientId { get; set; }
		[ForeignKey("ClientId")]
		public virtual Client? Client { get; set; }

		public Offer()
		{

		}
	}
}

File: be/nodejs/models/offer.js
'use strict';
const {
    Model
} = require('sequelize');
module.exports = (sequelize, DataTypes) => {
    // class offer extends Model {
    //     /**
    //      * Helper method for defining associations.
    //      * This method is not a part of Sequelize lifecycle.
    //      * The `models/index` file will call this method automatically.
    //      */
    //     static associate(models) {
    //         // define association here
    //         offer.hasOne(models.user_offer, { foreignKey: 'role_id' })
    //     }
    // }
    const offer = sequelize.define("offer", {
        Id: {
            type: DataTypes.CHAR,
            primaryKey: true,
            allowNull: false,
        },
        TotalPayment: {
            type: DataTypes.DECIMAL(65, 30),
            allowNull: false,
        },
        StartDate: {
            type: DataTypes.DATE,
            allowNull: false,
        },
        EndDate: {
            type: DataTypes.DATE,
            allowNull: false,
        },
        ProvideService: {
            type: DataTypes.TEXT('long'),
            allowNull: false,
        },
        EmployeeId: {
            type: DataTypes.CHAR,
            allowNull: false,
        },
        ClientId: {
            type: DataTypes.CHAR,
            allowNull: false,
        },
        CreateDate: {
            type: DataTypes.DATE,
            allowNull: false,
        },
        DeleteDate: {
            type: DataTypes.DATE
        },
        UpdatedDate: {
            type: DataTypes.DATE
        }
    }, {
        sequelize,
        timestamps: false,
        modelName: 'offer',
        tableName: 'offer',
    });
    offer.associate = (models) => {
        // define association here
        offer.belongsTo(models.client, { foreignKey: 'ClientId' })
        offer.belongsTo(models.employee, { foreignKey: 'EmployeeId' })
    }
    return offer;
};